README.md

```diff
@@ -161,6 +161,34 @@ will be added as they are required.
 
 ---
 
+`should_not_redirect`
+
+  + Use this method when you need to validate that a given request is not redirected
+
+**Usage:**
+
+  should\_not\_redirect TEST\_CASE\_NUMBER SOURCE\_PARAMETERIZED\_URL 
+
+**Example:**
+
+	should_not_redirect 0001 http://${targetSite}/original/location
+
+---
+
+`should_rewrite`
```

Comment:
I will look through my test cases and try to explain the reason for the should_not_redirect function.  It may be so specific to my needs that it doesn't have a larger audience.  More soon...

As for the mega commit, I see what you mean.  I am still wrapping my head around Git.

I will try to get back to you before the end  of the week.

Sent from my Windows Phone
________________________________
From: Jesse Mauntel<mailto:notifications@github.com>
Sent: ‎4/‎22/‎2015 10:41 PM
To: jmauntel/irule-tester<mailto:irule-tester@noreply.github.com>
Cc: Abend, Jon<mailto:jabend@cars.com>
Subject: Re: [irule-tester] Added two new functions and updated two existing to extend framework (#1)


In README.md<https://github.com/jmauntel/irule-tester/pull/1#discussion_r28934785>:

> @@ -161,6 +161,34 @@ will be added as they are required.
>
>  ---
>
> +`should_not_redirect`
> +
> +  + Use this method when you need to validate that a given request is not redirected
> +
> +**Usage:**
> +
> +  should\_not\_redirect TEST\_CASE\_NUMBER SOURCE\_PARAMETERIZED\_URL
> +
> +**Example:**
> +
> +     should_not_redirect 0001 http://${targetSite}/original/location
> +
> +---
> +
> +`should_rewrite`


Yep, I'm all on on this one. I'll merge it in.

BTW, when making multiple changes to a codeline with the intent on performing a pull request, it's much easier for the code maintainer to accept individual features if each feature has it's own commit. In this fashion, I could accept the addition of should_rewrite even when I have questions on should_not_redirect. When all the changes are in one big super-commit, it's an all or nothing accept.

The only way I can accept the parts of the pull request that I agree with and don't have remaining questions on is to make the changes myself, which I can do, but I'd prefer you get credit for the work you did and have your name set as the owner of the code change.

—
Reply to this email directly or view it on GitHub<https://github.com/jmauntel/irule-tester/pull/1/files#r28934785>.